metricflow-semantics/requirements-files/requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Always support a range of production DSI versions capped at the next breaking version in metricflow-semantics.
 # This allows us to sync new, non-breaking changes to dbt-core without getting a version mismatch in dbt-mantle,
 # which depends on a specific commit of DSI.
-dbt-semantic-interfaces>=0.8.4, <0.9.0
```

Comment:
I think it would be better to pin to >=0.9.0, <0.10.0 since there's a breaking change between 0.8 and 0.9 - but might not matter in practice since we pin to a more specific version in core?

Reply:
Ah you'll be blocked on that without making some changes in this repo. I can make that update after my current meeting

Reply:
🤷 to my mind if this code is compatible with earlier versions then it's okay to keep the pin looser but you have more context on this than I

Reply:
Agreed but it is unfortunately not compatible!

Reply:
updated!

Reply:
Sorry meeting ran very long! There are some other changes that need to be included based on the deps situation in this repo, working on them now